src/app/shared/stack-blitz/stack-blitz-writer.ts

```diff
@@ -203,7 +203,7 @@ export class StackBlitzWriter {
         .replace(/material-docs-example/g, data.selectorName)
         .replace(/\${title}/g, data.description);
     } else if (fileName === '.stackblitzrc') {
-      fileContent = fileContent.replace(/\${startCommand}/, isTest ? 'turbo test' : 'turbo start');
+      fileContent = fileContent.replace(/\${startCommand}/, isTest ? 'npm test' : 'npm start');
```

Comment:
We're using `yarn` in the examples, not `npm`.

Reply:
Oh thanks for pointing that out! That'll makes the PR much simpler then.